frontend/src/containers/UserListing.tsx

```diff
@@ -55,8 +128,11 @@ function UserListing() {
             },
             {
               Header: "Role",
-              accessor: "",
-              Cell: () => "Admin",
+              accessor: "roles",
+              Cell: (props: any) =>
+                props.value && props.value.length
+                  ? (props.value as Array<UserRoles>).join(",")
+                  : "Admin",
```

Comment:
I think we should display an empty value here instead of defaulting to `Admin`. Because defaulting to "Admin" gives a false perception that there is a role assigned, when it is not true.